Source/WebCore/svg/animation/SVGSMILElement.cpp

```diff
@@ -120,7 +120,7 @@ bool ConditionEventListener::operator==(const EventListener& listener) const
 void ConditionEventListener::handleEvent(ScriptExecutionContext&, Event&)
 {
     if (RefPtr animation = m_animation.get())
-        animation->handleConditionEvent(m_condition);
+        m_animation->addInstanceTime(m_condition->m_beginOrEnd, m_animation->elapsed() + m_condition->m_offset);
```

Comment:
should be `animation` here not `m_animation`

Reply:
Ops! Sorry! Missed it. Done in [55a14ae](https://github.com/WebKit/WebKit/commit/55a14aeac70bcb36689231445845e4bbfb6b9eb7).